contracts/stake/src/state.rs

```diff
@@ -263,19 +265,23 @@ impl StakeState {
 
     /// Rewards a `account` with the given `value`. If a stake does not exist
```

Comment:
```suggestion
    /// Rewards multiple accounts with the associated values based on the `rewards` being passed. If a stake does not exist
```

Now that account & value are no longer params, but encapsulated & consolidated in the rewards, we should make the comments a bit clearer. Added a suggestion for this

Reply:
@herr-seppia has commited to changing this in a future PR, which will implement event emission. I agree I could have done a better job, but for now I'm merging.